beta/src/components/MDX/Sandpack/CustomPreset.tsx

```diff
@@ -99,7 +99,7 @@ export function CustomPreset({
                     className="inline mr-1.5 text-xl"
                     displayDirection={isExpanded ? 'up' : 'down'}
                   />
-                  {isExpanded ? 'Show less' : 'Show more'}
+                  {isExpanded ? '숨기기' : '더보기'}
```

Comment:
```suggestion
                  {isExpanded ? '숨기기' : '더 보기'}
```
https://www.korean.go.kr/front/onlineQna/onlineQnaView.do?mn_id=216&qna_seq=208207&pageIndex=1